rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -1105,6 +1105,80 @@ void setLastRetiredFederationP2SHScript() {
 
     }
 
+    @ParameterizedTest
+    @MethodSource("provideFederationBtcUTXOsTestArguments")
+    void saveNewFederationBtcUTXOs_utxosShouldBeSavedToStorage(FederationStorageIndexKey federationStorageIndexKey, NetworkParameters networkParameters, ActivationConfig.ForBlock activations) {
+
+        StorageAccessor storageAccessor = new InMemoryStorage();
+        Address btcAddress = BitcoinTestUtils.createP2PKHAddress(networkParameters, "test");
+
+        int originalAmountOfUtxos = 2;
+        // Save utxos directly in storage
+        List<UTXO> expectedUtxos = BitcoinTestUtils.createUTXOs(originalAmountOfUtxos, btcAddress);
+        storageAccessor.saveToRepository(federationStorageIndexKey.getKey(), BridgeSerializationUtils.serializeUTXOList(expectedUtxos));
+
+        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+
+        // Getting utxos from method
+        List<UTXO> actualUtxos = federationStorageProvider.getNewFederationBtcUTXOs(networkParameters, activations);
+
+        // Should be as the expected utxos
+        assertEquals(actualUtxos.size(), originalAmountOfUtxos);
```

Comment:
WDYT if we put in the right side, the actual result, and the expected result on the left side of the assertions? This could be applied to all assertions that exist in the PR.

Reply:
Updated.